.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/asottile/add-trailing-comma
-    rev: v2.4.0
+    rev: v3.1.0
```

Comment:
It seems that ruff can do this: https://docs.astral.sh/ruff/rules/missing-trailing-comma/

Reply:
Also, lower in this file, should `--py37-plus` be changed to `--py38-plus` given that 3.8 is the oldest tested version?